cmd/server.go

```diff
@@ -307,6 +309,14 @@ var boolFlags = map[string]boolFlag{
 		defaultValue: false,
 	},
 }
+
+var stringSliceFlags = map[string]stringSliceFlag{
```

Comment:
Let's use a normal string and comma separate it when it comes in. That's the pattern for whitelist.